content/sensu-go/6.2/observability-pipeline/observe-entities/monitor-external-resources.md

```diff
@@ -459,6 +490,10 @@ sensu-docs          check-http                           0   false      2019-01-
 
 ## Next steps
 
+The files you created with check and entity definitions can become part of your [monitoring as code][4] repository.
+Storing your Sensu configurations the same way you would store code means they are portable and repeatable.
+Monitoring as code makes it possible to move to a more robust deployment without losing what you’ve started here and reproduce one environment’s configuration in another.
```

Comment:
Need to refine this when we have the monitoring as code message in better shape.